README.md

```diff
@@ -1,8 +1,106 @@
-# sdkms-pgp
+# sq-sdkms
 
 This crate leverages [sdkms-client-rust][sdkms-client-rust] and
-[Sequoia-PGP][Sequoia] to perform OpenPGP operations with keys stored in the
-[Fortanix Self-Defending Key-Management System][SDKMS].
+[Sequoia-PGP][Sequoia] to perform sensitive OpenPGP operations with keys stored
+in the [Fortanix Self-Defending Key-Management System][SDKMS].
+
+## Motivation
+
+Sequoia-PGP defines the [Signer][sequoia::Signer] and
+[Decryptor][sequoia::Decryptor] traits for low-level cryptographic operations
+with secret key material, and abstracts over these traits for PGP operations.
+This crate implements these traits with secrets stored inside Fortanix SDKMS,
+allowing to produce PGP material and decrypt ciphertexts without the need to
+export secrets bits.
```

Comment:
I'd reformulate this last sentence as: "This crate implements these traits for key material stored inside Fortanix SDKMS, enabling the production of PGP material and decryption of ciphertexts without the need to export any secrets."